amundsen_application/static/webpack.dev.ts

```diff
@@ -1,4 +1,27 @@
+import * as path from 'path';
 import merge from 'webpack-merge';
-import commonConfig from './webpack.common'
+import commonConfig, {htmlWebpackPluginConfig} from './webpack.common';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 
-export default merge(commonConfig, {});
+export default merge(commonConfig, {
+  devServer: {
+      contentBase: path.join(__dirname, '../build/lib/amundsen_application/static/dist'),
+      publicPath: '/static/dist/',
+      watchContentBase: true,
+      port: 60480,
+      proxy: {
+          '!(/static/dist/**.*)': {
+              target: 'http://127.0.0.1:60477',
+              secure: false,
+          },
+      },
+  },
+  output: {
```

Comment:
This configuration for `output` already exists in the common config, do we need it here?

Reply:
It's because I'm overwriting the plugins configuration from the common config so that it will attach a different port.